### PR TITLE
Add trees

### PR DESCRIPTION
**Pull request by ethanalee**

### CODE REVIEW

File: src/components/Home/Home.vue [hunk 1]
```diff
@@ -6,14 +6,20 @@
       <div id="cloud-2" class="clouds left-right"></div>
       <div id="cloud-3" class="clouds left-right"></div>
       <div id="cloud-4" class="clouds left-right"></div>
-      <div ></div>
+
+      <div></div>
```
Comment (jamiepinheiro) [inline src/components/Home/Home.vue]: Do we need this empty div?

File: src/components/Home/Home.vue [hunk 2]
```diff
@@ -60,114 +66,169 @@ export default Vue.extend({
...
-    }
-  }
-
-  MailSignup {
-    position: absolute;
-    bottom: 0;
-  }
+body::-webkit-scrollbar {
+  display: none;
+}
```
Comment (jamiepinheiro) [inline src/components/Home/Home.vue]: Can we actually leave this indented once? All the other files are following this structure.

File: src/components/Home/HowItWorks.vue [hunk 3]
```diff
@@ -24,31 +36,46 @@ export default Vue.extend({
...
+        ]
       ]
-    }
+    };
   }
 });
 </script>
 
 <style>
-
```
Comment (jamiepinheiro) [inline src/components/Home/HowItWorks.vue]: Same as before (if u feel strongly, please change all the files :) )

File: src/components/Home/Information.vue [hunk 4]
```diff
@@ -17,37 +30,134 @@ export default Vue.extend({
...
+  background-repeat: no-repeat;
+  background-size: contain;
+  overflow: hidden;
+}
+
+.tree-1 {
+  background-image: url("~@/assets/img/website/tree-1.svg");
+  position: absolute;
+  top: 102%;
+  right: 103%;
```
Comment (jamiepinheiro) [inline src/components/Home/Information.vue]: Can we move these to styles.css? Or maybe a new file trees.css

File: src/components/Home/HowItWorks.vue (hunk 3 above)
Comment (ethanalee) [reply src/components/Home/HowItWorks.vue]: oh sorry its the auto-linter

File: src/components/Home/Information.vue (hunk 4 above)
Comment (ethanalee) [reply src/components/Home/Information.vue]: yep got it

File: src/components/Home/Home.vue [hunk 5]
```diff
@@ -105,36 +111,36 @@ export default Vue.extend({
 
   #cloud-1 {
     background-image: url("~@/assets/img/website/cloud-1.svg");
-    animation-duration: 80s;
+    animation-duration: 50s;
```
Comment (jamiepinheiro) [inline src/components/Home/Home.vue]: Looks like bad merge here, can we revert to the values on master

File: src/components/Home/HowItWorks.vue (hunk 3 above)
Comment (jamiepinheiro) [reply src/components/Home/HowItWorks.vue]: all good haha

File: src/components/Home/Information.vue [hunk 6]
```diff
@@ -1,53 +1,163 @@
...
+    height: 70px;
+  }
+
+  .tree-9 {
+    background-image: url("~@/assets/img/website/tree-4.svg");
+    position: absolute;
+    top: -8%;
+    right: 100%;
+    width: 50px;
+    height: 50px;
```
Comment (jamiepinheiro) [inline src/components/Home/Information.vue]: trees.css maybe?

**Pull request merged**